cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,49 @@
 /// <reference types='cypress' />
 
-describe('Student Registration page', () => {
+describe('Student registration page', () => {
   before(() => {
-    
-  });
+    let user;
 
+    beforeEach(() => {
+    cy.visit('https://demoqa.com/automation-practice-form');
+    cy.task('generateUser').then(generatedUser => {
+      user = generatedUser;
+    });
+    });
+ 
   it('', () => {
-    
+
+  it('Fill all fields in forms except "picture"', () => {
+    cy.viewport(550, 750);
+    // cy.findByPlaceholder('First Name').type(user.firstName);
+    cy.get('#firstName').type(user.firstName);
+    // cy.findByPlaceholder('Last Name').type(user.lastName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.findByType('radio').check(user.gender, { force: true });
+    // cy.findByPlaceholder('Mobile Number').type(user.mobileNumber);
+    cy.get('#userNumber').type(user.mobileNumber);
```

Comment:
Consider using `cy.findByPlaceholder` for better readability and maintainability.